### PR TITLE
Locktime & proposalTime error label fix

### PR DESCRIPTION
**Pull request by MiltonTulli**
# Description
- Add conditional flag to display (or not) X icon on `NumericalInput` component to avoid founded issue.
Screen Shot 2022-11-21 at 14 00 41

- Display proper error label on `lockTime` and `proposalTime` fields for setGuildConfig Actions. Only display error for updated field, not both.
Screen Shot 2022-11-21 at 14 19 38

Closes https://github.com/DXgovernance/DAVI/issues/427

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] My code follows the existing style of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any UI changes have been tested and made responsive for mobile views

### CODE REVIEW

File: src/components/primitives/Forms/NumericalInput.tsx [hunk 1]
```diff
@@ -55,7 +57,13 @@ export const NumericalInput: React.FC<InputProps<string>> = ({
       maxLength={79}
       spellCheck="false"
       disabled={disabledState}
-      iconRight={<IconRight {...iconRightProps} />}
+      iconRight={
+        iconRight ? (
```
Comment (rulfo71) [inline src/components/primitives/Forms/NumericalInput.tsx:61]: just checking if iconRight is null is not enough?

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -271,6 +271,7 @@
   "noValueHasBeenUpdatedFromCurrentGuildConfig": "No value has been updated from current guild config",
   "proposalTimeHasToBeMoreThanZero": "Proposal time has to be more than 0",
   "lockTimeHasToBeHigherOrEqualToProposalTime": "Lock time has to be higher or equal to proposal time",
+  "proposalTimeHasToBeLowerOrEqualToLockTime": "Proposal time has to be lower or equal to Lock time",
```
Comment (rulfo71) [inline src/locales/en/translation.json:274]:
```suggestion
  "proposalTimeHasToBeLowerOrEqualToLockTime": "Proposal time has to be lower than or equal to the Lock time",

```

**Review by rulfo71 — APPROVED** [review]

**Comment by Filipv95** [comment]
@MiltonTulli Looks good to me!

File: src/locales/en/translation.json (hunk 2 above)
Comment (MiltonTulli) [reply src/locales/en/translation.json]: Done!

**Review by rulfo71 — APPROVED** [review]

**Pull request merged**